src/win/fs.c

```diff
@@ -1699,6 +1706,83 @@ static void fs__readlink(uv_fs_t* req) {
 }
 
 
+static size_t fs__realpath_handle(HANDLE handle, char** realpath_ptr) {
```

Comment:
Easier to close handle once outside the function, instead of the 5 different return options.
I prefer it that way.